packages/edit-post/src/components/header/header-toolbar/index.js

```diff
@@ -85,10 +85,10 @@ function HeaderToolbar() {
 	/* translators: accessibility text for the editor toolbar */
 	const toolbarAriaLabel = __( 'Document tools' );
 
-	const toggleListView = useCallback(
-		() => setIsListViewOpened( ! isListViewOpen ),
-		[ setIsListViewOpened, isListViewOpen ]
-	);
+	const toggleListView = useCallback( () => {
+		toggleFeature( 'showListViewByDefault' );
```

Comment:
Well toggleFeature means, this is still a preference. The only difference is that the preference is not used for the "initial state" but for the "latest state". I think that's not great because it doesn't behave like a preference and because it triggers an API request each time you open/close the list view.

Reply:
Sorry about the delay in replying, this one slipped off my list. Yeh, the use of preference here is confusing. If anything it should work the same as the pinned items approach used for the right panels.

Can you see any reason why the list view shouldn't work the same as the right inspector panel, ie. it doesn't have any preference, it just remembers its last state? /cc @jameskoster

Reply:
> Can you see any reason why the list view shouldn't work the same as the right inspector panel

Not really. The 'remembering' seems more useful / consistent.

Iirc the preference was originally created so that third parties can force List View to be open initially on new sites. As long as there is a way to handle that programmatically, then it would seem reasonable to remove the preference once the remembering is added.